packages/flutter_tools/lib/src/base/context.dart

```diff
@@ -116,7 +119,13 @@ class AppContext {
   /// Gets the value associated with the specified [type], or `null` if no
   /// such value has been associated.
   T get<T>() {
-    dynamic value = _generateIfNecessary(T, _overrides);
+    // Convert lookups of old process manager into new process manager.
```

Comment:
Does this need a TODO to track removing after the internal usage is sorted out?

Reply:
depends on if it works or not :)